indico/modules/events/operations.py

```diff
@@ -278,3 +280,41 @@ def unlock_event(event):
     db.session.flush()
     logger.info('Event %r unlocked by %r', event, session.user)
     event.log(EventLogRealm.event, EventLogKind.change, 'Event', 'Event unlocked', session.user)
+
+
+def create_reviewing_question(event, question_model, wtf_field_cls, form, data=None):
+    new_question = question_model()
+    new_question.no_score = True
+    if 'no_score' in form.data:
+        new_question.no_score = form.data['no_score']
```

Comment:
`if form.no_score:` and (in the next line) `form.no_score.data`. no need to build the data dict thrice

Reply:
`form.no_score` will raise an `AttributeError` if there is no such a property

Reply:
`if hasattr(form, 'no_score'):` will be a better option

Reply:
You can use `'no_score' in form`